nvflare/job_config/fed_job_config.py

```diff
@@ -385,3 +387,9 @@ def _get_deploy_map(self):
             deploy_map[app_name] = deploy_map.get(app_name, [])
             deploy_map[app_name].append(site)
         return deploy_map
+
+    def _trim_whitespace(self, string: str):
```

Comment:
Can this be done simply by: string.strip()?

Reply:
same as above